src/Raven.Server/Documents/Subscriptions/AbstractSubscriptionStorage.cs

```diff
@@ -137,7 +137,14 @@ public bool TryEnterSubscriptionsSemaphore()
 
     public void ReleaseSubscriptionsSemaphore()
     {
-        _concurrentConnectionsSemiSemaphore.Release();
+        try
+        {
+            _concurrentConnectionsSemiSemaphore.Release();
+        }
+        catch
+        {
+            //
```

Comment:
please write here something, or catch the expected exception

Reply:
Done